tutorials/tutorial_views.py

```diff
@@ -0,0 +1,944 @@
+"""
+    mss.tutorials.tutorial_views
+    ~~~~~~~~~~~~~~~~~~~
+
+    This python script generates an automatic demonstration of how to use the top view, side view, table view and
+    linear view section of Mission Support System in creating a project and planning the flightrack.
+
+    This file is part of mss.
+
+    :copyright: Copyright 2021 Hrithik Kumar Verma
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+import pyautogui as pag
+import multiprocessing
+import sys
+from sys import platform
+from pyscreeze import ImageNotFoundException
+from tutorials import screenrecorder as sr
+from mslib.msui import mss_pyui
+
+
+def initial_ops():
+    """
+    Executes the initial operations such as closing all opened windows and showing the desktop.
+    """
+    pag.sleep(5)
+    if platform == "linux" or platform == "linux2":
+        pag.hotkey('winleft', 'd')
+        print("\n INFO : Automation is running on Linux system..\n")
+    elif platform == "darwin":
+        pag.hotkey('option', 'command', 'm')
+        print("\n INFO : Automation is running on Mac OS..\n")
+    elif platform == "win32":
+        pag.hotkey('win', 'd')
+        print("\n INFO : Automation is running on Windows OS..\n")
+    else:
+        pag.alert(text="Sorry, no support on this platform!", title="Platform Exception", button='OK')
+    print("\n INFO : We will be learning how to create a project in MSS with all the views.\n")
+
+
+def call_recorder():
+    """
+    Calls the screen recorder class to start the recording of the automation.
+    """
+    sr.main()
+
+
+def call_mss():
+    """
+    Calls the main MSS GUI window since operations are to be performed on it only.
+    """
+    mss_pyui.main()
+
+
+def automate_views():
+    """
+    This is the main automating script of the MSS views tutorial which will cover all the views(topview, sideview,
+    tableview, linear view) in demonstrating how to create a project. This will be recorded and savedto a file having
+     dateframe nomenclature with a .mp4 extension(codec).
+    """
+    # Giving time for loading of the MSS GUI.
+    pag.sleep(5)
+
+    if platform == 'linux' or platform == 'linux2' or platform == 'darwin':
+        dir_path = 'pictures/tutorial_views/win/'
```

Comment:
there could be in utils a pictures functions, similiar to the icons functions which sets the path and also returns the needed image

Reply:
I will try to do that.